docs/recipes/nvhpc-a100-with-cuda-and-mpi-example.md

```diff
@@ -0,0 +1,267 @@
+---
+tags:
+ - Engaging
+ - Satori
+ - Howto Recipes
+ - nvhpc
+ - mpi
+ - cuda
+ - cuda aware mpi
+ - GPU
+ - Rocky Linux
+---
+
+# Example of a minimal program using the nvhpc stack with CUDA aware MPI
+
+## About NVHPC
+
+NVHPC is an integrated collection of software tools and libraries distributed by NVidia. An overview document describing nvhpc 
+can be found [here](https://developer.nvidia.com/hpc-sdk){.target=_blank}.
```

Comment:
Since open in new tab is being handled in PR #19 we can remove the `{.target=_blank}`.